synapse/lib/stormtypes.py

```diff
@@ -103,9 +103,13 @@ def addLibFuncs(self):
         ints = [intify(x) for x in vals]
         return max(*ints)
 
-    async def _print(self, mesg):
+    async def _print(self, mesg, **kwargs):
         if not isinstance(mesg, str):
             mesg = repr(mesg)
+        elif kwargs:
+            for name, valu in kwargs.items():
```

Comment:
can we functionalize this so both print and format use the same code; instead of copying it?